spanner/api/Spanner.Samples/AlterSequencePostgresql.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_postgresql_alter_sequence]
+
+using Google.Cloud.Spanner.Admin.Database.V1;
+using Google.Cloud.Spanner.Common.V1;
+using Google.Cloud.Spanner.Data;
+using Google.LongRunning;
+using System;
+using System.Threading.Tasks;
+
+public class AlterSequencePostgresqlSample
+{
+    public async Task AlterSequencePostgresqlSampleAsync(string projectId, string instanceId, string databaseId)
+    {
+        DatabaseAdminClient databaseAdminClient = new DatabaseAdminClientBuilder
+        {
+            Endpoint = "staging-wrenchworks.sandbox.googleapis.com"
+        }.Build();
+        DatabaseName databaseName = DatabaseName.FromProjectInstanceDatabase(projectId, instanceId, databaseId);
+        string[] statements =
+        {
+            "ALTER SEQUENCE Seq SKIP RANGE 1000 5000000;"
+        };
+        var operation = await databaseAdminClient.UpdateDatabaseDdlAsync(databaseName, statements);
+        var completedResponse = await operation.PollUntilCompletedAsync();
+        if (completedResponse.IsFaulted)
+        {
+            throw completedResponse.Exception;
+        }
+
```

Comment:
Can we add print statements here to show DDL update operation success? This is to maintain consistency across libraries.

Reply:
To clarify further, this would have print statements like "Altered Seq sequence to skip an inclusive range between 1000 and 5000000" (not the DDL statements).